db/migrate/20200611190026_create_departments.rb

```diff
@@ -1,9 +1,6 @@
 class CreateDepartments < ActiveRecord::Migration[6.0]
   def up
-    create_table :departments do |t|
-
-      t.timestamps
-    end
+    create_table :departments, &:timestamps
```

Comment:
😅 this is a pretty weird notation for migrations

Reply:
Hmmmm... I didnt do this